core/embed/sec/optiga/optiga.c

```diff
@@ -357,40 +384,92 @@ bool optiga_set_metadata(uint16_t oid, const optiga_metadata *metadata) {
   return true;
 }
 
-static bool optiga_pin_init_metadata(void) {
+static bool optiga_pin_init_metadata(optiga_ui_progress_t ui_progress) {
   optiga_metadata metadata = {0};
 
+  ui_progress();
+
   // Set metadata for counter-protected PIN secret.
   memzero(&metadata, sizeof(metadata));
   metadata.change = OPTIGA_META_ACCESS_ALWAYS;
-  metadata.read = ACCESS_STRETCHED_PIN;
+  metadata.read = ACCESS_LAST_STRETCHED_PIN;
   metadata.execute = OPTIGA_META_ACCESS_ALWAYS;
   metadata.data_type = TYPE_AUTOREF;
   if (!optiga_set_metadata(OID_PIN_SECRET, &metadata)) {
     return false;
   }
 
-  // Set metadata for stretched PIN.
+  ui_progress();
+
+#if OPTIGA_STRETCHED_PINS == 1
```

Comment:
This symbol doesn't seem to exist. Probably `STRETCHED_PIN_COUNT`.

Reply:
Fixed in https://github.com/trezor/trezor-firmware/pull/5845/commits/8a1c754f2faad77a20f6795fa7970f1ae0a8f7c7.